packages/dapp/src/components/ThemeSwitch/ThemeSwitch.tsx

```diff
@@ -43,6 +44,7 @@ export const ThemeSwitch = () => {
 
   return (
     <Tooltip
+      disableHoverListener={popperOpened}
```

Comment:
So many options! `disableHoverListener` works for the most part, but honestly I am not sure if any of those ones may be needed as well. Wdyt? @chybisov @abhishek-lifi @0xsebn 

```
      disableInteractive={popperOpened}
      disableFocusListener={popperOpened}
      disableTouchListener={popperOpened}
```

Reply:
[disableHoverListener](https://mui.com/material-ui/api/tooltip/#Tooltip-prop-disableHoverListener) seems like it should cover all cases.

I guessing this shouldn't be a valid case in phones anyways.

so, LGTM.

Reply:
Perfect!